src/arch/src/aarch64/fdt.rs

```diff
@@ -299,26 +299,28 @@ fn generate_prop64(cells: &[u64]) -> Vec<u8> {
 }
 
 // Following are the auxiliary function for creating the different nodes that we append to our FDT.
-fn create_cpu_nodes(fdt: &mut Vec<u8>, vcpu_mpidr: &Vec<u64>) -> Result<()> {
+fn create_cpu_nodes(fdt: &mut Vec<u8>, vcpu_mpidr: &[u64]) -> Result<()> {
     // See https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/arm/cpus.yaml.
     append_begin_node(fdt, "cpus")?;
     // As per documentation, on ARM v8 64-bit systems value should be set to 2.
     append_property_u32(fdt, "#address-cells", 0x02)?;
     append_property_u32(fdt, "#size-cells", 0x0)?;
-    let num_cpus = vcpu_mpidr.len();
 
-    for cpu_index in 0..num_cpus {
+    let more_than_one = vcpu_mpidr.len() > 1;
```

Comment:
`more_tan_one` sounds a bit confusing to me. I would rename it to `has_multiple_vcpus` or I would keep the old variable `num_cpus = vcpu_mpidr.len();`

Reply:
Done! Take a look!